can/interfaces/slcan.py

```diff
@@ -157,10 +161,12 @@ def __init__(
 
         super().__init__(channel, **kwargs)
 
-    def set_bitrate(self, bitrate: int) -> None:
+    def set_bitrate(self, bitrate: int, dbitrate: int = 0) -> None:
```

Comment:
I'd prefer `data_bitrate`  over `dbitrate` for consistency with other interfaces. The default value should be `None`, not zero.